storm-core/src/jvm/org/apache/storm/scheduler/SchedulerAssignment.java

```diff
@@ -55,4 +56,6 @@
     public Set<ExecutorDetails> getExecutors();
     
     public Set<WorkerSlot> getSlots();
+
+    public Map<WorkerSlot, Collection<ExecutorDetails>> getSlotToExecutors();
```

Comment:
very minor: The interface is public, so it's redundant to mark each method declaration as public.